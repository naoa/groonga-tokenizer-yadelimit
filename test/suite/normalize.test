register tokenizers/yadelimit

tokenize TokenYaDelimitCaret "AbCDï¼¤" NormalizerAuto
