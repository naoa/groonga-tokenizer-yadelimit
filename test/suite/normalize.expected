register tokenizers/yadelimit
[[0,0.0,0.0],true]
tokenize TokenYaDelimitCaret "AbCDï¼¤" NormalizerAuto
[[0,0.0,0.0],[{"value":"abcdd","position":0}]]
